float64 pos_x
float64 pos_y
float64 pos_z
float64 zyx_angle_z
float64 zyx_angle_y
float64 zyx_angle_x
bool hardcoded_wrist
---
float64 theta1
float64 theta2
float64 theta3
float64 theta4
float64 theta5
float64 theta6
